packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -36,23 +69,23 @@ class TestHostComponent {
 	@ViewChild(StarkDropdownComponent)
 	public dropdownComponent: StarkDropdownComponent;
 	public dropdownId: string;
-	public formControl: FormControl;
 	// public header?: string;
-	public disabled?: boolean;
-	public multiSelect?: string;
+	public disabled: boolean;
+	public multiSelect?: boolean;
 	public optionIdProperty?: string;
 	public optionLabelProperty?: string;
 	public options: any[];
 	public placeholder: string;
 	public value: any | any[];
 	public selectionChanged: Function;
-	public required?: boolean;
+	public required: boolean;
 }
 
 describe("DropdownComponent", () => {
```

Comment:
I think the structure of these unit tests is a bit messy. In the image below you can see marked in red what is not consistent and also an "orphan" `it`:

![dropdown unit tests structure](https://user-images.githubusercontent.com/5698318/54676929-21c3ab00-4b02-11e9-9559-c01e77e53455.PNG)

Reply:
It should be better now 😊